modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -381,7 +386,8 @@ public TableManager(
             SchemaSyncService schemaSyncService,
             CatalogService catalogService,
             HybridTimestampTracker observableTimestampTracker,
-            PlacementDriver placementDriver
+            PlacementDriver placementDriver,
+            Supplier<IgniteSql> sql
```

Comment:
the approach say us the we have cyclic dependencies.... It looks bad

Reply:
We already have cyclic dependencies in `ComputeExecutorImpl`.
We can pass `Ignite` instance instead of `Supplier<IgniteSql>`. Will it look better?

Reply:
Unfortunately with current approach it can't be fixed. I perhaps we rethinking the approach and remove SQL here in the future